recipes/flakeheaven/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "flakeheaven" %}
+{% set version = "0.11.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flakeheaven-{{ version }}.tar.gz
+  sha256: 20f5a573b8e85be5a73fed2e3967f7ab8b5e77714a5898d8b634dd31a0b75f9b
+
+build:
+  entry_points:
+    - flake8heavened = flakeheaven:flake8_entrypoint
+    - flakeheaven = flakeheaven:entrypoint
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.0,<4.0
```

```suggestion
    - python >=3.5
```
Ref: https://github.com/flakeheaven/flakeheaven/blob/3f0afc179dbd0907230fb3123f2938b0b63d3b7f/pyproject.toml#L66

Also, the upper bound of <4 is specified by poetry but shouldn't be set here because we prefer handling that wil conda-forge tools when the time comes.